<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #192231;
color: #D3D3D3;
border-bottom: 2px solid #4C5C75;
border-radius: 30px;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="menu_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0A1628;
color: #FFFFFF;
border-bottom: 3px solid #00FF7F;
border-radius: 30px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Pictures/proj QT/logo football.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string>Competition</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: white;</string>
    </property>
    <property name="text">
     <string>Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../Pictures/proj QT/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(40, 40, 40);
border: 2px solid #4C5C75;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4C5C75;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4C5C75;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>1071</width>
      <height>661</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
    border: 2px solid #2c2f3f;
    background: #1e1f29;
    border-radius: 5px;
}

QTabBar::tab {
    background: #1e1f29;
    color: #ffffff;
    padding: 8px 16px;
    border: 1px solid #2c2f3f;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:selected {
    background: #2c2f3f;
    color: #ffffff;
    font-weight: bold;
    border-bottom: 2px solid #0078d7;
}

QTabBar::tab:hover {
    background: #323546;
    color: #ffffff;
}

QTabBar::tab:!selected {
    margin-top: 2px;
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add Player</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>-10</y>
        <width>651</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0F111A;
color: #FFFFFF;
font-family: 'Segoe UI', Arial, sans-serif;
font-size: 12px;
border-radius: 15px;
padding: 15px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Date_Of_Birth</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>First_Name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDateEdit" name="dateEdit">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Last_Name</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLineEdit" name="lineEdit_9">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Jersey_Nb</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QComboBox" name="comboBoxTeam">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
         <item>
          <property name="text">
           <string>RMD</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QToolButton" name="toolButton">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 3px;
color: white;
margin-left: 15px;
margin-right:15px;
margin-bottom:15px;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Nationality</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_14">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Insert player photo</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QPushButton" name="button1">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(50, 150, 250, 0.8);
color: white;
font-size: 12px;
border-radius: 5px;
padding: 6px 12px;
margin-left:500px;
margin-right:20px;
margin-bottom:10px;</string>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Team</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QComboBox" name="comboBoxPosition">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
         <item>
          <property name="text">
           <string>Goalkeeper</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>470</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Display Player</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #1E1E2D;  /* Dark matte background */


 

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    background-color: #141821;  /* Slightly lighter for contrast */
    border: 2px solid #cccccc;  /* Keep borders subtle */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: white;</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>520</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
border-radius: 5px ;</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonModify">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>520</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #192231;
border-radius: 5px ;</string>
      </property>
      <property name="text">
       <string>Modify</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>511</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonViewDetails">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>520</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #192231;
border-radius: 5px ;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
     </widget>
     <widget class="QFrame" name="formFrame">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>130</y>
        <width>461</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
